fetch.bs

```diff
@@ -4272,6 +4273,11 @@ steps:
   </ol>
  </li>
 
+ <li><p>If <var>request</var> is a <a>subresource request</a> and <var>request</var>'s
+ <a for=request>window</a> is an <a>environment settings object</a>, then
+ <a>queue a fetch task</a> to run <span>process subresource link headers</span> given
```

Comment:
"process subresource link headers" is not defined anywhere. Relatedly, span does not cause cross-linking in Bikeshed specs.

Reply:
I am going to do an overhaul of this PR, ignore for now.

Reply:
Any idea when this will be overhauled?

I'm planning to do a follow-up change: I'd like to pass to the "process subresource link headers" algorithm whether the initiator subresource is render-blocking or not, so that we can create a render-blocking chain. It will depend on how this PR proceeds, though.

Reply:
I will get to it once https://github.com/whatwg/html/pull/7866 is in.
Also, I'm not sure how well tested the current support for `blocking` is in link headers, and I'm sure it's not spec'ed for early hints. I'm not sure if you wanted your follow up to include early hints, but if you do, first early hints have to support `blocking`. I would suggest to have the WPTs ready for those use-cases before we get to subresources.

Reply:
I'll file an issue for render-blocking chain, and also see how it should work with early hints. Thanks for the note!

Reply:
I filed whatwg/html#7899

Regarding early hints: I think `blocking` should be ignored on early hints, because there's no document to block when early hints are processed. It suffices as long as the actual response makes the document block on the early-hint-preloaded resources.